pkg/server/server.go

```diff
@@ -29,15 +29,22 @@ func NewServer(port int, logger logging.Logger, dataController *controllers.Data
 }
 
 // Start starts the http server
-func (s *Server) Start() {
+func (s *Server) Start(started chan bool) {
 	routers := s.createRouters()
 	s.logger.Infof("Listening on %d", s.port)
+	started <- true
 	err := http.ListenAndServe(fmt.Sprintf(":%d", s.port), s.logRequest(routers))
 	if err != nil {
 		s.logger.Error(err)
+		started <- false
 	}
 }
 
+// Stop stops the server
+func (s *Server) Stop() {
+	s.logger.Debug("Server stopped")
+}
+
```

Comment:
This doesn't do anything, actually. Is it really necessary to add it? We don't actually have an http server running :eyes: 

You can even take a look at the [babeltower](https://github.com/CESARBR/knot-babeltower/blob/master/pkg/server/server.go) server file as reference, but personally I believe we simply shouldn't add this method.

Reply:
@lcbm are you sure there isn't an HTTP server running? 👀
I agree with you this method doesn't do anything, but it'd be nice to clean up gracefully when stopping the server, i.e. @esdrasjnr can follow the reference suggested.